CONTRIBUTORS.md

```diff
@@ -6,7 +6,7 @@ The following people have contributed to Chapel releases:
 Contributors to the Chapel 1.31.0 release
```

Comment:
make sure to bump version before merge

Reply:
Done, thanks!  (I'm thinking I should just take the mention of the version out given how often I forget this... but didn't do it yet).